src/components/icons/svg/AudioIcon.js

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import Svg, { Path } from 'react-native-svg';
+
+export default function AudioIcon({ color, colors, ...props }) {
```

Comment:
```suggestion
export default function AudioIcon({ color = colors.black, colors, ...props }) {
```

yes, it really works :)

Reply:
... but I guess this needs to be reverted anyway